usecases/traverser/explorer.go

```diff
@@ -232,6 +232,23 @@ func (e *Explorer) searchResultsToGetResponse(ctx context.Context,
 			res.Schema.(map[string]interface{})["_additional"] = certainty
 		}
 
+		for _, selectProp := range params.Properties {
+			for _, refClass := range selectProp.Refs {
+				ref := res.Schema.(map[string]interface{})[selectProp.Name]
+				if ref != nil {
+					for _, innerRefProp := range ref.([]interface{}) {
+						innerRef := innerRefProp.(search.LocalRef)
```

Comment:
I think we need to actually check if this type assertion is correct. There is a case where you'd run into something else:

Assume the following Schema
- Three Classes: Car, ParkingLot, Garage
- A car can be parked either in a ParkingLot or in a Garage
- This would be reflected in the following schema prop: `{"name": "parkedAt", "dataType":["Garage", "ParkingLot"]}`
- If the user sets both target classes in their select props all is fine and both would be represented as `search.LocalRef`
- However, if the user wants only one resolved, the other one would still be present as an unresolved class and thus be a `models.MutlipleRef`. In that case we'd run into a panic

If it's not a `search.LocalRef` we can simply skip it, so adding:

```golang
innerRef, ok := innerRefProp.(search.LocalRef)
if !ok {
  continue
}
```

should be enough, I think